tests/import_test.go

```diff
@@ -18,6 +20,9 @@ const (
 	namespacePrefix                  = "importer"
 	assertionPollInterval            = 2 * time.Second
 	controllerSkipPVCCompleteTimeout = 60 * time.Second
+	bedEndpoint                      = "http://gopats.com/who-is-the goat.iso"
```

Comment:
Should this be 'badEndpoint'? Also you have a space in the URL.

Reply:
oops for both. thanks.